bedrock/base/templates/includes/banners/fundraiser.html

```diff
@@ -9,33 +9,52 @@
 {% block banner_id %}fundraising-banner{% endblock %}
 
 {% block banner_title %}
-  <span>{{ ftl('banner-fundraising-title') }}</span>
+  {% if switch('fundraising-banner-giving-tuesday-2021') %}
+    {{ ftl('banner-fundraising-title-giving-day', fallback='banner-fundraising-title') }}
+  {% else %}
+    {{ ftl('banner-fundraising-title-v2', fallback='banner-fundraising-title') }}
+  {% endif %}
 {% endblock %}
 
 {% block banner_content %}
-  <p>{{ ftl('banner-fundraising-body') }}</p>
+  {% if ftl_has_messages('banner-fundraising-body-v2', 'banner-fundraising-body-extra') %}
+    <p>{{ ftl('banner-fundraising-body-v2') }}</p>
+    <p>{{ ftl('banner-fundraising-body-extra') }}</p>
+  {% else %}
+    <p>{{ ftl('banner-fundraising-body') }}</p>
+  {% endif %}
 
-  <form id="fundraiser-form" class="c-fundraiser-form" method="get" action="https://donate.mozilla.org/{{ LANG }}/">
+  <form id="fundraiser-form" class="c-fundraiser-form" method="get" action="https://donate.mozilla.org/">
     <fieldset class="c-fundraiser-recurring">
-      <label for="monthly">
-        <input type="radio" value="monthly" id="monthly" name="frequency"> {{ ftl('banner-fundraising-monthly') }}
-      </label>
-
       <label for="onetime">
-        <input type="radio" value="single" id="onetime" name="frequency" checked> {{ ftl('banner-fundraising-one-time') }}
+        <input type="radio" value="single" id="onetime" name="frequency"> {{ ftl('banner-fundraising-one-time') }}
       </label>
-    </fieldset>
 
-    <fieldset class="c-fundraiser-options">
-    {% for amount in donate_params.preset_list %}
-      <label for="donate{{ amount }}"{% if amount == donate_params.default %} class="selected"{% endif %}>
-        <input class="c-fundraiser-amount-input" type="radio" value="{{ amount }}" id="donate{{ amount }}" name="amount"{% if amount == donate_params.default %} checked="checked"{% endif %}>
-        <span class="mzp-c-button mzp-t-secondary mzp-t-dark">{{ ftl('banner-fundraising-amount', amount=amount) }}</span>
+      <label for="monthly">
+        <input type="radio" value="monthly" id="monthly" name="frequency" checked> {{ ftl('banner-fundraising-monthly') }}
```

Comment:
[Late-breaking change request](https://github.com/mozilla/bedrock/issues/10700#issuecomment-972258794) to set the default back to One-time